control_unix.go

```diff
@@ -8,13 +8,20 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// This value has been taken from https://reviews.freebsd.org/D11003 since this is not yet provided in golang.
+const FREEBSD_SO_REUSEPORT_LB = 0x00010000
```

Comment:
No need to redefine this, this is define as `unix.SO_REUSEPORT_LB`.

Reply:
Interesting, as i could not find it when i was looking in the unix system package. That could be because i was looking in the macOS/linux one.
Will check with BSD package and see.